modules/portworx/scripts/px-sc.sh

```diff
@@ -3,7 +3,7 @@
 echo "creating storage classes"
 
 # CouchDB (Implemented application-level redundancy)
-cat <<EOF | oc create -f -
+cat <<EOF | kubectl create -f -
```

Comment:
Should this be "kubectl apply"?

See https://stackoverflow.com/questions/52500442/difference-between-kubectl-apply-and-kubectl-create

Reply:
In this case, create and apply would work without problems. However I believe, create would make the most sense as we don't necessarily want the users to overwrite the predefined storage classes by accident which an apply will happily overwrite.